pkg/machinery/config/types/extensionservicesconfig/extension_services_config.go

```diff
@@ -49,6 +49,7 @@ type V1Alpha1 struct {
 type ExtensionServiceConfig struct {
 	ExtensionName               string                       `yaml:"name"`
 	ExtensionServiceConfigFiles []ExtensionServiceConfigFile `yaml:"configFiles"`
+	ExtensionServiceEnvironment []string                     `yaml:"environment"`
```

Comment:
something is not wired properly, as I don't see the config reference documentation for this new config type